xml/docu_styleguide.structure.xml

```diff
@@ -1070,8 +1070,7 @@ will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>
     </listitem>
     <listitem>
      <para>
-      Avoid editing screenshots. If you need to edit a screenshot, use the
-      <phrase role="programname">Shutter</phrase> application. To anonymize
+      Avoid editing screenshots. To anonymize
       portions of a screenshot, use the <guimenu>Pixelize</guimenu> tool. To
       highlight parts of a screenshot, use the <guimenu>Rectangle</guimenu>
       tool or the <guimenu>Arrow</guimenu> tool. Never add callouts, text or
```

Comment:
The rest of the para builds on this recommendation of Shutter, so needs an edit too:

```suggestion
      Avoid editing screenshots. To anonymize
      portions of a screenshot, use a pixelization tool. To
      highlight parts of a screenshot, use a rectangle
      tool or an arrow tool. Never add callouts, text or
      freely drawn objects. Always select colors that provide a good
      contrast with their background.
```